tests/tensor/random/test_op.py

```diff
@@ -22,20 +22,24 @@ def set_aesara_flags():
         yield
 
 
-def test_default_shape_from_params():
+def test_default_supp_shape_from_params():
```

Comment:
Add docstring please, even if its simple

Reply:
This seems like quite a burden request for a PR that only intended to change the name of the function.

Reply:
I went and merged the PR. If you think the tests warrant docstrings/ comments please open an issue for that

Reply:
Thanks Ricardo